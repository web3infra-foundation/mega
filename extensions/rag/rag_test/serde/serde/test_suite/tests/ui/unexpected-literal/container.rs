use serde_derive::Serialize;

#[derive(Serialize)]
#[serde("literal")]
struct S;

fn main() {}
