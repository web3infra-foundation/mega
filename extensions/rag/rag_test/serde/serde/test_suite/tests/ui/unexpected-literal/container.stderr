error: unexpected literal in nested attribute, expected ident
 --> tests/ui/unexpected-literal/container.rs:4:9
  |
4 | #[serde("literal")]
  |         ^^^^^^^^^
